### PR TITLE
docs: vanity url top directory

### PR DESCRIPTION
**Pull request by lxfontes**
Prepares `docs` directory for Github Pages.

Once we are in the new repo, we need to:
- Create `docs/CNAME` file
- Enable gh pages
- Enable 'Force HTTPS'

### CODE REVIEW

**Comment by ydnar** [comment]
Great!

- Does the CNAME file live in the `go-imports` directory?
- Should we add a README.md file to this directory describing its purpose?

**Comment by lxfontes** [comment]
It should go in the directory 👍 
just added a readme too. I wasn't sure if gh-pages would prefer the readme over index.html, but seem it does prefer index.html

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# go.bytecodealliance.org Vanity URLs
```
Comment (ydnar) [inline docs/README.md:1]: Can you replace "vanity URLs" with "Go import paths" in this document?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+     ```html
+     <html>
+     	<head>
+     		<meta name="go-import" content="go.bytecodealliance.org/foo git https://github.com/bytecodealliance/go-foo" />
+     	</head>
+     </html>
+     ```
+
+NOTE: Vanity URLs must be unique and not shadow existing package names in this repository. For ex: Creating a vanity URL `go.bytecodealliance.org/cm` would shadow the `cm` package in this repository.
```
Comment (ydnar) [inline docs/README.md:22]: `For example:`

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+This directory hosts the configuration for Go vanity URLs under `go.bytecodealliance.org` domain.
+
+More information about vanity URLs can be found in the [Go documentation](https://golang.org/cmd/go/#hdr-Remote_import_paths).
+
+## Adding a new vanity URL
+
+If you want to create a vanity URL `go.bytecodealliance.org/foo` pointing to `github.com/bytecodealliance/go-foo`:
+
+- Create directory `foo`
```
Comment (ydnar) [inline docs/README.md:11]: Maybe say `docs/foo` here and on the next line?

**Comment by ydnar** [comment]
Going to make a few small changes…

**Pull request merged**